modules/tensor_mechanics/src/actions/DomainIntegralAction.C

```diff
@@ -252,10 +288,10 @@ DomainIntegralAction::act()
     params.set<bool>("use_displaced_mesh") = _use_displaced_mesh;
     if (_integrals.count(INTERACTION_INTEGRAL_T) != 0)
     {
-      params.set<VariableName>("disp_x") = _disp_x;
-      params.set<VariableName>("disp_y") = _disp_y;
-      if (_disp_z != "")
-        params.set<VariableName>("disp_z") = _disp_z;
+      params.set<VariableName>("disp_x") = _displacements[0];
```

Comment:
I don’t think these are used anymore, are they?

Reply:
CrackFrontDefinition userobject still uses disp_x, disp_y and disp_z. I have not edited that file yet.

I can either edit it now or change the CrackFrontDefinition and add EshelbyTensor automatically through the DomainIntegralAction later in a seperate pull request.

Reply:
OK, let's address that later.